src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -1928,6 +1928,60 @@ std::set<std::vector<element::Type>> jit_sigmoid_emitter::get_supported_precisio
     return {{element::f32}};
 }
 
+/// SOFT_SIGN ///
+jit_soft_sign_emitter::jit_soft_sign_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const std::shared_ptr<ov::Node>& node)
+        : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+}
+
+jit_soft_sign_emitter::jit_soft_sign_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const ov::element::Type exec_prc) : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+}
+
+size_t jit_soft_sign_emitter::get_inputs_count() const { return 1; }
+
+size_t jit_soft_sign_emitter::get_aux_vecs_count() const { return 2; }
+
+size_t jit_soft_sign_emitter::get_aux_gprs_count() const { return 1; }
+
+void jit_soft_sign_emitter::emit_impl(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+void jit_soft_sign_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (exec_prc_ != ov::element::f32) {
+        OPENVINO_THROW("unsupported precision: " + exec_prc_.to_string());
+    }
+
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    const TReg src(in_vec_idxs[0]);
+    const TReg dst(out_vec_idxs[0]);
+    const TReg aux1(aux_vec_idxs[0]);
+    const TReg aux2(aux_vec_idxs[1]);
+
+    h->fabs(aux1.s, src.s);
+    h->ld1r(aux2.s, table_val2("one"));
+    h->fadd(aux1.s, aux1.s, aux2.s);
+    h->fdiv(dst.s, src.s, aux1.s);
```

Comment:
@eshoguli @dmitry-gorokhov I thought this should have been enough to implement softsign (x / (1 + |x|)) but when I run tests via the smoke command I am getting failures in SoftSign activation. 
<img width="1439" alt="Screenshot 2024-10-05 at 12 14 54 PM" src="https://github.com/user-attachments/assets/2e63390f-a19e-4bbe-a45d-24580a0a4f29">

Could you guys please review the changes once?

Reply:
Hi @nashez, tests are failing because of kernel type mismatch check (the implementation itself is correct). Please update the condition to fix expected kernel type on tests side: https://github.com/openvinotoolkit/openvino/blame/master/src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp#L198

Reply:
Oh, I see you already added it. Then please try to find out the reason why the test still sets expected prim type as ref_f32

Reply:
There was an issue with my build of the tests on my local machine. Fixed and the tests are all passing now. 
This is good to merge @dmitry-gorokhov